commons/src/main/java/esa/commons/logging/EncoderImpl.java

```diff
@@ -44,6 +45,11 @@
     private static final Function<StringBuilder, byte[]> ENCODER;
     private static final BiFunction<StringBuilder, Charset, byte[]> ENCODER_WITH_CHARSET;
 
+    // See https://github.com/oracle/graal/blob/master/sdk/src/org.graalvm.nativeimage/src/org/graalvm/nativeimage/
+    // ImageInfo.java
+    private static final boolean RUNNING_IN_NATIVE_IMAGE = SystemPropertyUtil.contains(
```

Comment:
do not use classes of netty which is not dependent directly

Reply:
done~